staging/src/k8s.io/genericapiserver/pkg/authentication/user/doc.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // Package user contains utilities for dealing with simple user exchange in the auth
 // packages. The user.Info interface defines an interface for exchanging that info.
-package user // import "k8s.io/kubernetes/pkg/auth/user"
+package user // import "k8s.io/genericapiserver/pkg/authentication/user"
```

Comment:
Gotcha. Is `k8s.io/genericapiserver` set in stone? Just thinking about hypothetical users seeing this in the repo list. `k8s.io/apiserver-lib`? Or just plain `k8s.io/apiserver` with the understanding that the specific binaries will all be prefixed with something (e.g., `kube-apiserver`)